addon/webextension/selector/ui.js

```diff
@@ -322,11 +322,98 @@ this.ui = (function() { // eslint-disable-line no-unused-vars
 
   iframePreSelection.onResize = watchFunction(onResize.bind(iframePreSelection));
 
+  let iframePreview = exports.iframePreview = {
+    element: null,
+    document: null,
+    sizeTracking: {
```

Comment:
Huh, I notice the `iframePreSelection` code has size tracking, but I wonder if it really needs it? Unlike selection, we just need an element that is the height and width of the viewport, position:fixed.  The selection iframe does something more complicated that this element won't have to do.